packages/autocomplete-js/src/__tests__/render.test.ts

```diff
@@ -536,6 +536,48 @@ describe('render', () => {
     });
   });
 
+  test('provides the scoped API', () => {
+    const container = document.createElement('div');
+    const panelContainer = document.createElement('div');
+
+    document.body.appendChild(panelContainer);
+    autocomplete<{ label: string }>({
+      container,
+      panelContainer,
+      initialState: {
+        isOpen: true,
+      },
+      getSources() {
+        return [
+          {
+            sourceId: 'testSource',
+            getItems() {
+              return [{ label: '1' }];
+            },
+            templates: {
+              item({ item }) {
+                return item.label;
+              },
+            },
+          },
+        ];
+      },
+      render(params, _root) {
```

Comment:
```suggestion
      render(params) {
```

Is it left on purpose?

Reply:
Yeah just to show that it's still passed, but both are fine to me.